ssg/build_yaml.py

```diff
@@ -260,7 +218,6 @@ class Benchmark(XCCDFEntity):
     """Represents XCCDF Benchmark
     """
     KEYS = dict(
-        title=lambda: "",
```

Comment:
Why is the Benchmark title removed?

Reply:
Because Benchmark is a child of XCCDFEntity as well, an will inherit it.